src/js/components/Form/index.d.ts

```diff
@@ -1,11 +1,16 @@
 import * as React from "react";
 
+export interface FormExtendedEvent extends React.FormEvent {
+  value: Record<string, unknown>;
```

Comment:
A Record<K, T> is an object type whose property keys are K and whose property values are T. Doesn't it imply that the `unknown` in this case should be `any` instead?

Reply:
Hi!
As far as I can understand `unknown` is a type-safe version of `any`. It says that “you can receive anything here and you should consider checking the type”.
But thinking about use-cases, I think you are actually right. It should be or `any` or the actual types. I will set it to any for now.